cities.go

```diff
@@ -0,0 +1,158 @@
+package main
+
+// Random names to prefix to deploy ids.
+// Makes it easier to talk about some specific deploy
+
+var CITIES = []string{
+	"tokyo",
+	"new-york",
+	"sao-paulo",
+	"seoul-south",
+	"mexico-city",
+	"osaka",
+	"manila",
+	"mumbai",
+	"delhi",
+	"jakarta",
+	"lagos",
+	"kolkata",
+	"cairo",
+	"los-angeles",
+	"buenos-aires",
+	"rio-de-janeiro",
+	"moscow",
+	"shanghai",
+	"karachi",
+	"paris",
+	"istanbul",
+	"nagoya",
+	"beijing",
+	"chicago",
+	"london",
+	"shenzhen",
+	"tehran",
+	"bogota",
+	"lima",
+	"bangkok",
+	"johannesburg",
+	"chennai",
+	"taipei",
+	"baghdad",
+	"santiago",
+	"bangalore",
+	"hyderabad",
+	"st-petersburg",
+	"philadelphia",
+	"lahore",
+	"kinshasa",
+	"miami",
+	"ho-chi-minh-city",
+	"madrid",
+	"tianjin",
+	"kuala-lumpur",
+	"toronto",
+	"milan",
+	"shenyang",
+	"dallas",
+	"boston",
+	"singapore",
+	"washington",
+	"detroit",
+	"barcelona",
+	"houston",
+	"athens",
+	"berlin",
+	"sydney",
+	"atlanta",
+	"san-francisco",
+	"montreal",
+	"melbourne",
+	"ankara",
+	"phoenix",
+	"seattle",
+	"cape-town",
+	"san-diego",
+	"rome",
+	"naples",
+	"minneapolis",
+	"tel-aviv",
+	"birmingham",
+	"frankfurt",
+	"lisbon",
+	"manchester",
+	"san-juan",
+	"baltimore",
+	"sapporo",
+	"tampa",
+	"taichung",
+	"warsaw",
+	"denver",
+	"cologne",
+	"hamburg",
+	"dubai",
+	"vancouver",
+	"beirut",
+	"budapest",
+	"cleveland",
+	"pittsburgh",
+	"harare",
+	"kuwait",
+	"munich",
+	"portland",
+	"brussels",
+	"vienna",
+	"san-jose",
+	"damman",
+	"copenhagen",
+	"brisbane",
+	"riverside",
+	"cincinnati",
+	"phnom-penh",
+	"leeds",
+	"stockholm",
+	"sacramento",
+	"kansas-city",
+	"marseille",
+	"turin",
+	"lyon",
+	"rotterdam",
+	"las-vegas",
+	"milwaukee",
+	"stuttgart",
+	"indianapolis",
+	"glasgow",
+	"perth",
+	"orlando",
+	"columbus",
+	"dublin",
+	"auckland",
+	"porto",
+	"new-orleans",
+	"adelaide",
+	"helsinki",
+	"buffalo",
+	"memphis",
+	"antwerp",
+	"austin",
+	"port-elizabeth",
+	"stamford",
+	"salt-lake-city",
+	"calgary",
+	"ottawa",
+	"richmond",
+	"penrith",
+	"bondi",
+	"bordeaux",
+	"nashville",
+	"oklahoma-city",
```

Comment:
Oh, not much, just thinking that words with hyphens in them are hard to say "amazing dash salt dash lake dash city", and the pattern for deploys is slightly harder to parse as you can't just take the first two elements of the split string.

Just a thought, I don't feel strongly about it :)